From 314bd53959d01c095580ee1dfbe7577c1ddb95b6 Mon Sep 17 00:00:00 2001
From: "Saravanan.K.S" <saravanan.kadambathursubramaniyam@windriver.com>
Date: Thu, 27 Mar 2025 03:00:12 +0000
Subject: [PATCH] Remove invalid header file inclusion

File "tegra-prod.h" was not present in Jetpack SDK hence removing it.

Upstream-Status: Inappropriate [WRLinux specific]

Signed-off-by: Saravanan.K.S <saravanan.kadambathursubramaniyam@windriver.com>
---
 nvidia-oot/include/linux/tegra-oot-prod.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/nvidia-oot/include/linux/tegra-oot-prod.h b/nvidia-oot/include/linux/tegra-oot-prod.h
index de0021c..992f981 100644
--- a/nvidia-oot/include/linux/tegra-oot-prod.h
+++ b/nvidia-oot/include/linux/tegra-oot-prod.h
@@ -10,7 +10,7 @@
 
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(5, 16, 0)
 #ifdef CONFIG_TEGRA_PROD_ENABLE
-#include <soc/tegra/tegra-prod.h>
+#include <linux/tegra-prod-upstream-dummy.h>
 #include <linux/tegra_prod.h>
 #else
 #include <linux/tegra-prod-upstream-dummy.h>
-- 
2.40.0

